doorstop/core/document.py

```diff
@@ -193,7 +193,11 @@ def _iter(self, reload=False):
                 else:
                     self._items.append(item)
                     if reload:
-                        item.load(reload=reload)
+                        try:
+                            item.load(reload=reload)
+                        except Exception as e:
+                            print("!! Exception loading %s" % item)
+                            raise e
```

Comment:
I believe this can simple be a bare `raise` statement and you can remove `as e` from above.